frontend/src/app/inbox/ChatPreview.tsx

```diff
@@ -36,6 +41,15 @@ export default function ChatPreview({ match, users, community }) {
                 </div>
                 <div className="ReleasePreview">
                     <p>{formatShortDate(release_timestamp)}</p>
+                    <div>
+                        {authUser?.uid === currentChat.latestMessage?.from ? (
+                            <FontAwesomeIcon icon={faCircleCheck} className="readIcon" />
+                        ) : (
+                            <div className="readIcon">
+                                <FontAwesomeIcon icon={faMoon} />
+                            </div>
+                        )}
+                    </div>
```

Comment:
You can it easier to read the JSX/HTML by saving the icon you want to a variable and then injecting it. Like this:

```tsx
// Save each icon as a variable
const iconResponded = <FontAwesomeIcon icon={faCircleCheck} className="readIcon" />
const iconNotResponded = <FontAwesomeIcon icon={faMoon} className="readIcon" />
// Save a variable for the condition
const hasResponded = authUser?.uid === currentChat.latestMessage?.from

// Save the chosen icon based on the condition to a variable
const iconRespondedStatus = hasResponded ? iconResponded : iconNotResponded

// Insert the chosen icon into the layout
return (
  ...
  <div>{iconRespondedStatus}</div>
  ...
)
```